pkg/services/network/constants.go

```diff
@@ -22,6 +22,11 @@ const (
 	NSXTTypeNetwork = "nsx-t"
 	// NSXTVNetSelectorKey is also defined in VM Operator.
 	NSXTVNetSelectorKey = "ncp.vmware.com/virtual-network-name"
+	// NSXTVPCSubnetSetNetworkType is the name of the NSX VPC network type. Please refer to:
+	// https://github.com/vmware-tanzu/vm-operator/blob/main/api/v1alpha1/virtualmachine_types.go#L149.
+	NSXTVPCSubnetSetNetworkType = "nsx-t-subnetset"
```

Comment:
Just to double check. This is the correct value? I saw "nsx-t-subnet" used in some document.

Reply:
Yes, "nsx-t-subnet" and "nsx-t-subnetset" are 2 similar network types and "nsx-t-subnetset"  has the ability of autoscale. We will only use "nsx-t-subnetset" for cluster network.